packaging/ubuntu-14.04/control

```diff
@@ -21,7 +21,7 @@ Build-Depends: autoconf,
                init-system-helpers,
                libapparmor-dev,
                libglib2.0-dev,
-               libseccomp-dev (>= 2.1.1-1ubuntu1~trusty3),
+               libseccomp-dev (>= 2.1.1-1ubuntu1~trusty4),
```

Comment:
libseccomp-dev in trusty does not have a static library. I added it for trusty and uploaded a new SRU to unblock this, see https://launchpad.net/ubuntu/+source/libseccomp/2.1.1-1ubuntu1~trusty4